sdk/monitor/Azure.Monitor.Ingestion/api/Azure.Monitor.Ingestion.netstandard2.0.cs

```diff
@@ -19,11 +19,20 @@ public enum ServiceVersion
             V2021_11_01_Preview = 1,
         }
     }
+    public partial class UploadFailedArgs : Azure.SyncAsyncEventArgs
+    {
+        internal UploadFailedArgs() : base (default(bool), default(System.Threading.CancellationToken)) { }
+        public System.Exception Exception { get { throw null; } }
+        public int FailedLogs { get { throw null; } }
+    }
```

Comment:
I thought we were going to add `IReadOnlyList<object> FailedLogs { get; }` to this as well so people could do things like attempt to resubmit if they wanted.  If we had that, we could also drop the count because it'd just be `FailedLogs.Count`.